app/src/main/res/layout/schedule_message_dialog.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/scheduled_message_dialog_holder"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:paddingVertical="@dimen/activity_margin">
+
+    <com.simplemobiletools.commons.views.MyTextView
+        android:id="@+id/subtitle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginHorizontal="@dimen/activity_margin"
+        android:background="?attr/selectableItemBackground"
+        android:includeFontPadding="false"
+        android:padding="@dimen/tiny_margin"
+        android:text="@string/schedule_send_warning"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.appcompat.widget.AppCompatImageView
+        android:id="@+id/dateIcon"
+        android:layout_width="@dimen/date_time_icon_size"
+        android:layout_height="@dimen/date_time_icon_size"
+        android:layout_marginStart="@dimen/big_margin"
+        android:layout_marginTop="@dimen/big_margin"
+        android:src="@drawable/ic_calendar_month_vector"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/subtitle" />
+
+    <com.simplemobiletools.commons.views.MyTextView
+        android:id="@+id/edit_date"
+        android:layout_width="wrap_content"
```

Comment:
make the whole line clickable, like we do everywhere

Reply:
_ripples_
<img src="https://user-images.githubusercontent.com/36371707/195930750-7b8857d8-5c14-48ec-83fb-9c98827d61c3.png" width=264 />